docs/get-details/dapps/smart-contracts/apps/index.md

```diff
@@ -768,15 +768,43 @@ Box refs can be added to the boxes array using `goal` or any of the SDKs.
     goal app method --app-id=53 --method="add_member2()void" --box="53,str:BoxA" --from=CONP4XZSXVZYA7PGYH7426OCAROGQPBTWBUD2334KPEAZIHY7ZRR653AFY
     ```
 
-=== "Python SDK"
+=== "Python"
     ```py
-    # Algorand Python SDK
-    # Using ATC
     atc = AtomicTransactionComposer()
-    atc.add_method_call(app_id, my_method, addr, sp, signer, method_args=[1,5], boxes=[[app_id, “key”]],
-    )
+    atc.add_method_call(app_id, my_method, addr, sp, signer, method_args=[1,5], boxes=[[app_id, “key”]])
+    ```
+
+=== "JavaScript"
+    ```js
+    const atc = new AtomicTransactionComposer();
+    atc.addMethodCall({
+        appID: appId,
+        method: myMethod,
+        methodArgs: [1,5],
+        boxes: [{ appIndex: appId, name: new Uint8Array(Buffer.from("key")) }],
+        sender: acct.addr,
+        suggestedParams: sp,
+        signer: algosdk.makeBasicAccountTransactionSigner(acct),
+    });
+    ```
+
+=== "Go"
+    ```go
+	var boxAtc = transaction.AtomicTransactionComposer{}
+	err := boxAtc.AddMethodCall(transaction.AddMethodCallParams{
+		AppID:           appId,
+		Sender:          acct.Address,
+		SuggestedParams: sp,
+		OnComplete:      types.NoOpOC,
+		Signer:          signer,
+        Method:          method,
+        BoxReferences:   types.AppBoxReference{AppID: appId, Name: []byte("key")},
+		MethodArgs:      []interface{}{1,5},
+	})
+    // ...
```

Comment:
Sure, the formatting won't matter when rendered in HTML, but I couldn't let myself approve it.
```suggestion
    var boxAtc = transaction.AtomicTransactionComposer{}
    err := boxAtc.AddMethodCall(transaction.AddMethodCallParams{
        AppID:           appId,
        Sender:          acct.Address,
        SuggestedParams: sp,
        OnComplete:      types.NoOpOC,
        Signer:          signer,
        Method:          method,
        BoxReferences:   types.AppBoxReference{AppID: appId, Name: []byte("key")},
        MethodArgs:      []interface{}{1,5},
    })
    // ...
```